src/stories/5-custom/filter/filter-checkbox-item.stories.tsx

```diff
@@ -95,5 +148,87 @@ FilterCheckBoxItem.args = {
 				{ label: "black", value: "black" },
 			],
 		},
+		filterItem3: {
+			label: "Collapsible item",
+			referenceKey: "filter-item-checkbox",
+			collapsible: true,
+			showDivider: true,
+			options: [
+				{ label: "red", value: "red" },
+				{ label: "blue", value: "blue" },
+			],
+		},
+		filterItem4: {
+			label: "Non-Collapsible item",
+			referenceKey: "filter-item-checkbox",
+			collapsible: false,
+			showDivider: false,
+			options: [
+				{ label: "red", value: "red" },
+				{ label: "blue", value: "blue" },
+			],
+		},
+		filterItem5: {
+			label: "Collapsible item with divider",
+			referenceKey: "filter-item-checkbox",
+			collapsible: true,
+			showDivider: true,
+			options: [
+				{ label: "red", value: "red" },
+				{ label: "blue", value: "blue" },
+			],
+		},
+		filterItem6: {
+			label: "Collapsible item with mobile divider",
+			referenceKey: "filter-item-checkbox",
+			collapsible: true,
+			showMobileDivider: true,
+			showDivider: false,
+			options: [
+				{ label: "red", value: "red" },
+				{ label: "blue", value: "blue" },
+			],
+		},
+	},
+};
+
+export const FilterCheckBoxItemWithDefaultValues = Template("wrapper-default-values").bind({});
+FilterCheckBoxItemWithDefaultValues.args = {
+	referenceKey: "filter",
+	defaultValues: ["red", "orange"],
+	children: {
+		"wrapper-default-values": {
+			label: "With 5 or more items",
+			referenceKey: "filter-item-checkbox",
+			options: [
+				{ label: "red", value: "red" },
+				{ label: "blue", value: "blue" },
+				{ label: "green", value: "green" },
+				{ label: "orange", value: "orange" },
+				{ label: "yellow", value: "yellow" },
+				{ label: "black", value: "black" },
+			],
+		},
 	},
 };
+
+// export const FilterCheckBoxItemWithValidation = Template("wrapper-validation").bind({});
```

Comment:
as mentioned, you can remove the validation story and its corresponding typings if it is not supported yet. 

as the filter components are custom components using the `referenceKey` (as opposed to the standard ones using `uiType`), you have more controls over what features you want to support in your components.

Reply:
This is commented out for now, so we can add it back when implementing validation. do you want us to remove commented out part as well?

Reply:
since validation is not supported, you can:
- remove the commented story
- remove validation by omitting it from `IFilterItemCheckboxSchema`
```
export interface IFilterItemCheckboxSchema<V = undefined>
	extends Omit<ICustomFieldJsonSchema<"filter-item-checkbox", V>, "validation"> {
	label: string;
	options: IOption[];
	collapsible?: boolean | undefined;
	showDivider?: boolean | undefined;
	showMobileDivider?: boolean | undefined;
}
```
- remove `setFieldValidationConfig` from `filter-item-checkbox`